### PR TITLE
Hide announcement in admin

### PR DESCRIPTION
**Pull request by sunny-viktoryia**
closes https://github.com/sparkletown/internal-sparkle-issues/issues/1583

or should I better remove the whole component? What approach is better here in your opinion? 🙃

### CODE REVIEW

File: src/components/organisms/AdminVenueView/components/RunTabView/RunTabView.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const RunTabView: React.FC<RunTabViewProps> = ({ venue }) => {
         </div>
       </AdminSidebar>
       <AdminShowcase className="RunTabView__main" variant="no-scroll">
-        <div className="RunTabView__toolbar RunTabView--spacing">
+        <div className="RunTabView__toolbar RunTabView--spacing mod--hidden">
```
Comment (goran-peoski-work) [inline src/components/organisms/AdminVenueView/components/RunTabView/RunTabView.tsx:42]: This should make it render only to people who have the flag set on their user.

```suggestion
        <TesterRestricted>
        <div className="RunTabView__toolbar RunTabView--spacing">
```

Alternatively, the "soft removal" (you can copy the approach from `disableSettings.ts`) is:

```suggestion
      {!DISABLED_DUE_TO_1583 &&
        <div className="RunTabView__toolbar RunTabView--spacing mod--hidden">
```

**Review by goran-peoski-work — APPROVED** [review]
LGTM.

Be it your first approach, or if you pick one of the suggested...

ping @colinhowe

**Comment by colinhowe** [comment]
Delete it. We'll use the more complex one rather than this one

**Comment by sunny-viktoryia** [comment]
@colinhowe, deleted. ✅

**Pull request merged**